js/everything.js

```diff
@@ -495,11 +495,11 @@ $(function() {
                     $("#search-results").show();
                     // Clear out results
                     $("#search-results ul").html("");
-
-                    var start = "<li><h4><a href=?v=";
+                    var start = "<li><h4><a style='float:right' href=?v=";
```

Comment:
Let's remove all the inline style and create a new CSS class, same for the results themselves